CHANGELOG.md

```diff
@@ -7,10 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [v.13.1] - 2019-07-22
+## [v1.13.2] - 2019-07-25
+
+- Re-release of v1.13.1 to fix broken npm packages.
+
+## [v1.13.1] - 2019-07-22
```

Comment:
Please put a note in to not use 1.13.1, but to upgrade to 1.13.2

Reply:
e197f0842